site/src/api/api.ts

```diff
@@ -1555,17 +1555,9 @@ export const getInsightsTemplate = async (
   return response.data;
 };
 
-export interface Health {
-  healthy: boolean;
-  time: string;
-  coder_version: string;
-  access_url: { healthy: boolean };
-  database: { healthy: boolean };
-  derp: { healthy: boolean };
-  websocket: { healthy: boolean };
-}
-
 export const getHealth = async () => {
-  const response = await axios.get<Health>("/api/v2/debug/health");
+  const response = await axios.get<TypesGen.HealthcheckReport>(
```

Comment:
@BrunoQuaresma Do you know why it uses `Health` instead of `TypesGen.HealthcheckReport`? Is there any strong reason why it does not use `api/api`?

Reply:
Extracted it here: https://github.com/coder/coder/pull/10650

Reply:
I don't remember but this is a good improvement for sure.